packages/bpk-component-infinite-scroll/README.md

```diff
@@ -38,13 +38,11 @@ const elementsArray = [
 ]
 
 const InfiniteList = withInfiniteScroll(SomeList);
+const dataSource = new ArrayDataSource(elementsArray);
```

Comment:
Now we can specialise here `ArrayDataSource<string>`

Reply:
I though we weren't using flow in the `README.md`. Added

Reply:
Didn't realise this was in the README.md yeah we should probably remove it?